.github/workflows/upload-test-stats.yml

```diff
@@ -54,7 +58,7 @@ jobs:
           HEAD_BRANCH: ${{ github.event.workflow_run.head_branch }}
         run: |
           echo "${WORKFLOW_URL}"
-          python3 -m tools.stats.upload_test_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}" --head-branch "${HEAD_BRANCH}"
+          python3 -m tools.stats.upload_test_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}" --changed-files ${{ steps.changed-files.outputs.all_changed_files }} --head-branch "${HEAD_BRANCH}"
```

Comment:
What does the path in the `changed-files` value look like? Is it relative to the repo root or does it have the full path on the machine?